modules/install/pages/install-swap-space.adoc

```diff
@@ -51,6 +51,7 @@ If the output doesn't match the <<recommended-swappiness-settings,recommended se
 
 . Change the swappiness setting.
 +
+[upperroman]
```

Comment:
This is the bulleted list style?
Is this part of our standard tone? I don't find `upperroman` elsewhere in docs-server

Reply:
No, that's Roman numbers for an ordered sublist. 

I'd better check the style guide and see what the form is for sublists.

Reply:
Sorted. Had a chat with Sarah, and I've left as the Antora default.